index.js

```diff
@@ -1,285 +0,0 @@
-//assets
-let dirt;
-let iron;
-let oil;
-let rock;
-let gold;
-let dug;
-
-let digSound;
-
-let playerLeft;
-let playerDown;
-let playerRight;
-let playerNorm;
-
-
-let gameRunning=false;
-let showStart = true;
-let resetButton;
-
-let startScreen;
-let gameOverScreen;
-let winScreen;
-
-let matrix;
-let mapArr;
-
-let ironCount= 0;
-let goldCount = 0;
-let fuel = 15;
-let player = {
-    x:300,
-    y:0
-}
-
-
-let playerDirection;
-
-
-
-function preload(){
-    dirt = loadImage(`img/dirt.png`);
-    gold = loadImage(`img/gold.png`);
-    iron = loadImage(`img/iron.png`);
-    oil = loadImage(`img/oil.png`);
-    rock = loadImage(`img/rock.png`);
-    dug = loadImage(`img/dug.png`);
-
-    //soundFormats('wav');
-    //digSound = new sound(`img/digging.wav`);
-
-    playerLeft = loadImage(`img/player-left.png`);
-    playerDown = loadImage(`img/player-down.png`);
-    playerRight = loadImage(`img/player-right.png`);
-    playerNorm = loadImage(`img/player-norm.png`);
-
-    startScreen = loadImage(`img/start-background-overlay - Copy.png`);
-    gameOverScreen = loadImage(`img/newGameOver-background.png`);
-    winScreen = loadImage(`img/winScreen.png`);
-}
-
-function setup(){
-    createCanvas(650,1500);
-    //generateMap(); 
-   makeMap();
-   
-
-}
-
-
-/*
```

Comment:
big commented out pieces of code can usually be deleted and committed with a nice commit message. 
If you need to get them back, revert that commit. 

Another thing you could do is to have that commit in a separate branch if it's experimental or something. But having big blocks of commented out code should not happen to often. Because that's what git is kind of for.